Sage/carbon
# S1T-33 Added grouped and size labels to the button-toggle

# Description
Added small and grouped options to the button-toggle. This is for the multicurrency toggle in the new invoices page.

# TODO

# Screenshots / Gifs
Grouped & Small:
![screen shot 2017-09-11 at 10 38 09](https://user-images.githubusercontent.com/20279205/30268409-81b36dbe-96dd-11e7-984e-33e1dd14a13b.png)

post-review:
![screen shot 2017-09-11 at 13 22 46](https://user-images.githubusercontent.com/20279205/30274226-6466526e-96f4-11e7-8c6e-22407fad2c9e.png)


Grouped & Large:
![screen shot 2017-09-11 at 10 39 56](https://user-images.githubusercontent.com/20279205/30268428-9453f33a-96dd-11e7-9ba9-5b3c3aa1cd50.png)

Ungrouped & Small:

![screen shot 2017-09-11 at 10 40 25](https://user-images.githubusercontent.com/20279205/30268450-a5190994-96dd-11e7-905a-62e787476a24.png)

# Testing Instructions
Check the different combinations of the new features, make sure the styling is correct and buttons can be selected.
Also check the styling with a disabled tag.
